src/data/usershistory.json

```diff
@@ -0,0 +1,33 @@
+
```

Comment:
the idea of this file is great but we can use a different mechanism to do that so in the future it will be deleted